packages/protocol/contracts/bridge/BridgeErrors.sol

```diff
@@ -14,7 +14,7 @@ abstract contract BridgeErrors {
     error B_ERC20_CANNOT_RECEIVE();
 
     /// Emitted when Ether has already been released as part of a transfer
-    error B_ETHER_RELEASED_ALREADY();
+    error B_TOKENS_RELEASED_ALREADY();
```

Comment:
Again, Bridge shall not know about "tokens", it knows about Ether. So please revert this change.